### PR TITLE
Fix typo `regitration` to `registration`

### PR DESCRIPTION
**Pull request by xuezhaojun**
Also updated in backplane-operator: https://github.com/stolostron/backplane-operator/pull/1077

### CODE REVIEW

File: build/setup-import-controller.sh [hunk 1]
```diff
@@ -170,7 +170,7 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: managedcluster-import-controller-agent-regitration-client
+  name: managedcluster-import-controller-agent-registration-client
```
Comment (zhujian7) [inline build/setup-import-controller.sh:173]: please test the upgrade case, if I remember correctly, the roleRef can not be updated.

**Review by zhujian7 — CHANGES_REQUESTED** [review]

File: build/setup-import-controller.sh (hunk 1 above)
Comment (zhujian7) [reply build/setup-import-controller.sh:173]: And for the change in the backplane operator https://github.com/stolostron/backplane-operator/pull/1077/files , why do we only need to update `ClusterRole`, how about the role binding, roleRef there?

Comment (xuezhaojun) [reply build/setup-import-controller.sh:173]: There is no rolebinding and roleRef for this clusterrole, the clusterrole is created for users who want to use agent-registration, they create their serviceaccount and clusterrolebinding. 

I put this clusterrole in backplane-operator only want to reduce a common clusterrole that users need to create. But users can create the clusterrole by themselves as well.

Comment (xuezhaojun) [reply build/setup-import-controller.sh:173]: We don't have roleRef need to updated, only this clusterrole.

**Pull request merged**